ashley/src/test/kotlin/ktx/ashley/EntitiesSpec.kt

```diff
@@ -81,5 +85,12 @@ object EntitiesSpec : Spek({
         assertThat(Texture.mapper !in entity).isTrue()
       }
     }
+
+    describe("add component function") {
```

Comment:
Add another test with a configuration block for `Transform` to check if the changes are correctly applied (i.e. change `x` and `y`  with `add(engine) { ... }` and check their values).